aws-athena/ctas/proximity-dist_pin_to_pin.sql

```diff
@@ -1,136 +1,2 @@
-            FROM (
-                SELECT
-                    dp.x_3435,
-                    dp.y_3435,
-                    loc.year,
-                    loc.pin10,
-                    ST_DISTANCE(ST_POINT(dp.x_3435, dp.y_3435), loc.point) AS dist
-                FROM distinct_pins AS dp
-                INNER JOIN pin_locations AS loc
-                    ON ST_CONTAINS(
-                        ST_BUFFER(ST_POINT(dp.x_3435, dp.y_3435), 1000), loc.point
-                    )
-            ) AS dists
-        )
-        WHERE row_num <= 4
-    )
-
-    SELECT *
-    FROM (
-        SELECT
-            pcl.pin10,
-            MAX(CASE
-                WHEN pd.row_num = 2 THEN pd.pin10
-            END) AS nearest_neighbor_1_pin10,
-            MAX(CASE
-                WHEN pd.row_num = 2 THEN pd.dist
-            END) AS nearest_neighbor_1_dist_ft,
-            MAX(CASE
-                WHEN pd.row_num = 3 THEN pd.pin10
-            END) AS nearest_neighbor_2_pin10,
-            MAX(CASE
-                WHEN pd.row_num = 3 THEN pd.dist
-            END) AS nearest_neighbor_2_dist_ft,
-            MAX(CASE
-                WHEN pd.row_num = 4 THEN pd.pin10
-            END) AS nearest_neighbor_3_pin10,
-            MAX(CASE
-                WHEN pd.row_num = 4 THEN pd.dist
-            END) AS nearest_neighbor_3_dist_ft,
-            pcl.year
-        FROM spatial.parcel AS pcl
-        INNER JOIN pin_dists AS pd
-            ON pcl.x_3435 = pd.x_3435
-            AND pcl.y_3435 = pd.y_3435
-            AND pcl.year = pd.year
-        GROUP BY pcl.pin10, pcl.year
-    )
-    WHERE nearest_neighbor_1_pin10 IS NOT NULL
-        AND nearest_neighbor_2_pin10 IS NOT NULL
-        AND nearest_neighbor_3_pin10 IS NOT NULL
-)
-
--- Consolidate unbucketed files into single files and delete temp table
-CREATE TABLE IF NOT EXISTS proximity.dist_pin_to_pin
-WITH (
-    format='Parquet',
-    write_compression = 'SNAPPY',
-    external_location='s3://ccao-athena-ctas-us-east-1/proximity/dist_pin_to_pin',
-    partitioned_by = ARRAY['year'],
-    bucketed_by = ARRAY['pin10'],
-    bucket_count = 1
-) AS (
-    SELECT
-        pin10,
-        nearest_neighbor_1_pin10,
-        nearest_neighbor_1_dist_ft,
-        nearest_neighbor_2_pin10,
-        nearest_neighbor_2_dist_ft,
-        nearest_neighbor_3_pin10,
-        nearest_neighbor_3_dist_ft,
-        year
-    FROM proximity.dist_pin_to_pin_temp
-    UNION
-    SELECT
-        pin10,
-        nearest_neighbor_1_pin10,
-        nearest_neighbor_1_dist_ft,
-        nearest_neighbor_2_pin10,
-        nearest_neighbor_2_dist_ft,
-        nearest_neighbor_3_pin10,
-        nearest_neighbor_3_dist_ft,
-        year
-    FROM proximity.dist_pin_to_pin_temp2
-);
-
-DROP TABLE IF EXISTS proximity.dist_pin_to_pin_temp
-DROP TABLE IF EXISTS proximity.dist_pin_to_pin_temp2
+-- View that finds the 3 nearest neighbor PINs for every PIN for every year
+SELECT * FROM {{ ref('proximity.dist_pin_to_pin_10km') }}

```

Comment:
**issue (blocking):** If `dist_pin_to_pin_10km.sql` only contains the values _missing_ from `dist_pin_to_pin_1km.sql`, then shouldn't this be the union of `dist_pin_to_pin_1km.sql` and `dist_pin_to_pin_10km.sql`?